src/consensus/params.h

```diff
@@ -39,6 +39,32 @@ struct BIP9Deployment {
     int64_t nThreshold;
 };
 
+enum LLMQType : uint8_t
+{
+    LLMQ_NONE = 0xff,
+
+    LLMQ_50_60 = 1, // 50 members, 30 (60%) threshold, one per hour
+    LLMQ_400_60 = 2, // 400 members, 240 (60%) threshold, one every 12 hours
+    LLMQ_400_85 = 3, // 400 members, 340 (85%) threshold, one every 24 hours
+
+    // for testing only
+    LLMQ_10_60 = 100, // 10 members, 6 (60%) threshold, one per hour
+};
+
+struct LLMQParams {
+    LLMQType type;
```

Comment:
Would be nice to have at least some brief description of these, smth similar to the way it's done for `BIP9Deployment` struct above.

Reply:
Done